spec/features/registration/sign_up_as_an_organisation_spec.rb

```diff
@@ -188,4 +188,16 @@
       expect(page).to have_content("Email is already associated with an account. If you can't sign in, reset your password")
     end
   end
+
+  context 'when no whitelisted organisation is selected' do
+    let(:non_whitelist_org_name) {'Not Whitelisted'}
+
+    before { sign_up_for_account }
+
+    it 'will tell the user the organisation is not whitelisted' do
+      update_user_details(organisation_name: non_whitelist_org_name)
```

Comment:
this amounts to not actually selecting an organisation name, as apposed to trying to set `Not Whitelisted` as the name.
This means the test is passing due to an indirect reason (blank is not whitelisted).

This is due to the conditional added in `sign_up_helpers`.
`expect(page).not_to have_content("Organisation name can't be blank")` will fail if this is the case.